types/jquery.notify/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for jquery.notify 1.5
+// Project: https://github.com/ehynds/jquery-notify (jQuery Notify UI Widget by Eric Hynds)
```

Comment:
I just based on its file name - `jquery.notify.js`.
Also I can see that there are more names like "jquery." than "jquery-" in the repo. 
But I can change if you think it should be with hyphen.